### PR TITLE
Add version history for 5.3.4

### PR DESCRIPTION
**Pull request by dgault**
See https://github.com/openmicroscopy/bioformats/milestone/48?closed=1
for list of included PRs.

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
 ===============
 
+5.3.4 (2017 February 21)
+------------------------
+
+Bug fixes:
+
+* ImageJ
+   - fix for a NullPointerException when exporting images that were not opened via 
+     theBio-Formats importer, and thus do not have a complete OMEXMLMetadata store.
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: missing space 'theBio-Formats'

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+
+* Java 1.9
+   - fix compile and runtime errors to enable building with Java 1.9
+
+Documentation improvements:
+
+* new public sample files added for ECAT7 (thanks to Torsten Stöter)
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: There's a format support fix associated with these samples too - https://github.com/openmicroscopy/bioformats/pull/2755

**Comment by sbesson** [comment]
See https://github.com/openmicroscopy/bioformats/milestone/48?closed=1 for the complete list of PRs. We should also document the bump of the `ome-model` component to 5.4.0.

**Comment by hflynn** [comment]
I'll rebuild staging once this passes Travis but it looks like it should be ok now. One minor question though - since this versioning of the model component is new, do we want to add a caveat in brackets that this isn't bumping the schema version?

**Comment by hflynn** [comment]
Rebuilding staging docs now

File: docs/sphinx/about/whats-new.txt [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+
+* Java 1.9
+   - fix compile and runtime errors to enable building with Java 1.9
+
+* ECAT7
+   - update to add support for different versions of ECAT7 files (thanks to Torsten Stöter)
+
+Updated build system:
+
+* updated dependency for `ome-model <https://github.com/ome/ome-model>` in the POM to
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: Sorry, this link needs `_` at the end to render as a hyper link - see https://www.openmicroscopy.org/site/support/bio-formats5.3-staging/about/whats-new.html

**Comment by hflynn** [comment]
One further fix needed I'm afraid, other than that, looks good to me now if @sbesson is also happy with the content.

File: docs/sphinx/about/whats-new.txt (hunk 3 above)
Comment (dgault) [inline docs/sphinx/about/whats-new.txt]: Is the underscore needed for external links?

Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: It's for any hyperlinks unless they are formatted as `:doc:` or internal references

Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: http://www.sphinx-doc.org/en/stable/rest.html#hyperlinks

**Comment by hflynn** [comment]
Sorry @dgault but that's not the link I meant - it's the external link to the model repo that needs fixing, ones set using the `:role:` markup don't need underscores.

File: docs/sphinx/about/whats-new.txt [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+
+* ECAT7
+   - update to add support for different versions of ECAT7 files (thanks to Torsten Stöter)
+
+Updated build system:
+
+* updated dependency for `ome-model <https://github.com/ome/ome-model>`_ in the POM to 
+  version 5.4.0. This allows for improved ROI handling by enabling support for Shape 
+  objects with Transform attributes. OME-XML schema version remains unchanged as 
+  :model_doc:`OME schema 2016-06 </schemas/june-2016-2.html>`
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: You also don't actually need the `/` in front of 'schemas/june-2016-2.html' but the link seems to be working anyway

File: docs/sphinx/about/whats-new.txt [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
 ===============
 
+5.3.4 (2017 February 21)
+------------------------
+
+Bug fixes:
+
+* ImageJ
+   - fix for a NullPointerException when exporting images that were not opened via 
+     the Bio-Formats importer, and thus do not have a complete OMEXMLMetadata store.
```
Comment (mtbc) [inline docs/sphinx/about/whats-new.txt]: not sure why this point is unusual in ending with a period

**Pull request merged**